tests/UI/campaigns/functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js

```diff
@@ -0,0 +1,244 @@
+require('module-alias/register');
+
+// Import utils
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+
+// Common tests login BO
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const carriersPage = require('@pages/BO/shipping/carriers');
+const addCarrierPage = require('@pages/BO/shipping/carriers/add');
+
+// Import data
+const CarrierFaker = require('@data/faker/carrier');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shipping_carriers_quickEditAndBulkActions';
```

Comment:
```suggestion
const baseContext = 'functional_BO_shipping_carriers_QuickEditAndBulkActions';
```

Reply:
Not sure that's how we write them ?

Reply:
Yes, it is a small error, I will change the name of the file :+1: 
The first letter must be lowercase

Reply:
Done thx :)